Dockerfile

```diff
@@ -1,6 +1,22 @@
-FROM debian:stable-slim
+FROM golang:1.14 AS build
 
-RUN apt-get update && \
-    apt-get -y install ca-certificates
```

Comment:
@tzununbekov  do we really  need `ca-certificates` ?

Reply:
yep, we need it to let `tm` binary communicate with external hosts over TLS, ie. right now image built "from scratch" complains about github certificates:
```
docker run -v /home/ubuntu/Downloads/config.json:/.kube/config --rm -ti tm deploy task -f https://raw.githubusercontent.com/triggermesh/knative-lambda-runtime/master/go-1.x/runtime.yaml

FATA[0000] task not found: Get "https://raw.githubusercontent.com/triggermesh/knative-lambda-runtime/master/go-1.x/runtime.yaml": x509: certificate signed by unknown authority 
```

Reply:
okay got it.

Reply:
restored back to debian:stable-slim + ca-certificates